coordinator/txmanager.go

```diff
@@ -288,32 +288,38 @@ func (t *TxManager) sendRollupForgeBatch(ctx context.Context, batchInfo *BatchIn
 // checkEthTransactionReceipt takes the txHash from the BatchInfo and stores
 // the corresponding receipt if found
 func (t *TxManager) checkEthTransactionReceipt(ctx context.Context, batchInfo *BatchInfo) error {
-	txHash := batchInfo.EthTx.Hash()
 	var receipt *types.Receipt
 	var err error
-	for attempt := 0; attempt < t.cfg.EthClientAttempts; attempt++ {
-		receipt, err = t.ethClient.EthTransactionReceipt(ctx, txHash)
-		if ctx.Err() != nil {
-			continue
-		} else if tracerr.Unwrap(err) == ethereum.NotFound {
-			err = nil
-			break
-		} else if err != nil {
-			log.Errorw("TxManager ethClient.EthTransactionReceipt",
-				"attempt", attempt, "err", err)
-		} else {
+	// taking receipt from last transaction
+	for i := len(batchInfo.EthTxs) - 1; i > 0; i-- {
```

Comment:
Is it okay, that I take the last transaction to get a receipt?

Reply:
Yes, because in this task we will not change the way hermeznode manage the nonce. 
So we just need to make sure this change does not break how hermeznode manage the nonces and tx receipts nowadays. 
This change seems to be correct for me but @ed255 's second opinion is very valid too.